### PR TITLE
fix: alignment of text inside the child element to that of Exapandabl…

### LINKED ISSUE #851: Exapndable List View in Faq Activity are not align with the child elements
**Summary:**

In Faq Activity, the Expandable ListView is not aligned with the child view text.

**Steps to reproduce:**

Navigate to FAQ section clicking on the overlay menu icon.expand the list by clicking on down arrow. 

**Expected behaviour:**

The child view after the expanding of ListView should be the same align to each other.

**Observed behaviour:**

Currently, the child views text that shows the answer is not aligned to the text of the ListView.

**Device and Android version:**

Android Version - Android 10
Android Device - POCO X2

**Screenshots:**

Screenshot_2020-03-23-12-37-08-333_org mifos mobilewallet mifospay

### PR DESCRIPTION
**Pull request by refactor-droidyy**
## Issue Fix
Fixes #851 

## Screenshots

Screenshot_2020-03-23-12-03-38-569_org mifos mobilewallet mifospay (1)

## Description

Adjust Margin 

##

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by refactor-droidyy** [comment]
@luckyman20 please review.

File: mifospay/src/main/res/layout/faq_list_child.xml [hunk 1]
```diff
@@ -8,7 +8,10 @@
         android:id="@+id/faq_list_child"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:layout_margin="16dp"
+        android:layout_marginLeft="@dimen/value_40dp"
+        android:layout_marginTop="@dimen/value_16dp"
+        android:layout_marginRight="@dimen/value_16sp"
```
Comment (jawidMuhammadi) [inline mifospay/src/main/res/layout/faq_list_child.xml:13]: Change the `layout_marginRight` value also to density-independent pixels (**dp**)

**Comment by refactor-droidyy** [comment]
@jawidMuhammadi please review.

**Comment by luckyman20** [comment]
Please check Issue #851

**Pull request closed without merging**